### ISSUE TITLE
Receptor start fail in latest codes

### ISSUE THREAD

**Issue by LaynePeng** [cloudfoundry-incubator/lattice#82]
Hi, I met a strange error after update my Lattice to the latest codes: the Receptor cannot start since initializeReceptorBBS, and the error message is: 

```
{"timestamp":"1428505821.944561958","source":"receptor","message":"receptor.fail
ed-parsing-consul-cluster","log_level":3,"data":{"error":"scheme must be http or
 https","trace":"goroutine 1 [running]:\ngithub.com/pivotal-golang/lager.(*logge
r).Fatal(0xc208046180, 0xa18210, 0x1d, 0x7f62bceb9b80, 0xc20802b5e0, 0x0, 0x0, 0
x0)\n\t/workspace/Diego/diego-release/src/github.com/pivotal-golang/lager/logger
.go:131 +0xc8\nmain.initializeReceptorBBS(0x7f62bcebeab0, 0xc208046180, 0x0, 0x0
)\n\t/workspace/Diego/diego-release/src/github.com/cloudfoundry-incubator/recept
or/cmd/receptor/main.go:248 +0x354\nmain.main()\n\t/workspace/Diego/diego-releas
e/src/github.com/cloudfoundry-incubator/receptor/cmd/receptor/main.go:150 +0x192
\n"}}
panic: scheme must be http or https
```

I found there are some updates on these part, but I cannot find how the consulCluster come. Can anyone give me some clue?

**Comment by LaynePeng** [comment]
No problem. I will wait for the update, and close the issue after it works.

**Comment by davidwadden** [comment]
We updated the Upstart scripts to support the latest diego-release, which landed on master with v0.2.2-22-gf5532f9.  Feel free to go ahead and pull the latest & greatest of both lattice and diego-release and let us know if you run into any issues.   Going to close this issue for now.
